src/components/pagination-nav/pagination-nav.js

```diff
@@ -0,0 +1,208 @@
+      el.disabled = false;
+      el.removeAttribute(this.options.attribActive);
+      this.anchorAttributesHelper(el);
+    });
+    if (selectorPageSelect) {
+      const selectorPageSelectOptions = selectorPageSelect.options;
+      Array.prototype.forEach.call(selectorPageSelectOptions, el => {
+        el.removeAttribute(this.options.attribActive);
+      });
+      if (!evt.target.matches(this.options.selectorPageSelect)) {
+        selectorPageSelect.classList.remove(this.options.classActive);
+        selectorPageSelect.value = '';
+      }
+    }
+  };
+
+  /**
+   * Add active state on click
+   */
+  handleClick = evt => {
+    let nextActivePageNumber = this.getActivePageNumber();
+    const pageElementNodeList = this.element.querySelectorAll(this.options.selectorPageElement);
+    const selectorPageSelect = this.element.querySelector(this.options.selectorPageSelect);
+    this.clearActivePage(evt);
+    if (evt.target.matches(this.options.selectorPageButton)) {
+      nextActivePageNumber = Number(evt.target.getAttribute(this.options.attribPage));
+    }
+    if (evt.target.matches(this.options.selectorPageDirection)) {
+      if (evt.target.getAttribute(this.options.attribPageDirection) === 'previous') {
+        nextActivePageNumber -= 1;
+      }
+      if (evt.target.getAttribute(this.options.attribPageDirection) === 'next') {
+        nextActivePageNumber += 1;
+      }
+    }
+    const pageTargetElement = pageElementNodeList[nextActivePageNumber - 1];
+    pageTargetElement.setAttribute(this.options.attribActive, true);
+    if (pageTargetElement.tagName === 'OPTION') {
+      selectorPageSelect.value = this.getActivePageNumber();
+      selectorPageSelect.classList.add(this.options.classActive);
+    } else {
+      pageTargetElement.classList.add(this.options.classActive, this.options.classDisabled);
+      pageTargetElement.disabled = true;
+      if (pageTargetElement.tagName === 'A') {
+        pageTargetElement.setAttribute('tabIndex', -1);
+        pageTargetElement.setAttribute('aria-disabled', true);
+      }
+    }
+    this.setPrevNextStates();
+  };
+
+  /**
+   * Handle select menu on change
+   */
+  handleSelectChange = evt => {
+    this.clearActivePage(evt);
+    const selectorPageSelect = this.element.querySelector(this.options.selectorPageSelect);
+    const selectorPageSelectOptions = selectorPageSelect.options;
+    selectorPageSelectOptions[selectorPageSelectOptions.selectedIndex].setAttribute(this.options.attribActive, true);
+    evt.target.classList.add(this.options.classActive);
+    this.setPrevNextStates();
+  };
+
+  /**
+   * Set Previous and Next button states
+   */
+  setPrevNextStates = () => {
+    const pageElementNodeList = this.element.querySelectorAll(this.options.selectorPageElement);
+    const totalPages = pageElementNodeList.length;
+    const pageDirectionElementPrevious = this.element.querySelector(`[${this.options.attribPageDirection}="previous"]`);
```

Comment:
Though this looks a neat technique, our codebase uses separate options for attributes/selectors. Doing the same here will be great.